tools/create-app.bash

```diff
@@ -5,10 +5,23 @@
 
 # Params: $1: output directory
 
-# Sign app for release
+### Assets ###
+
+# Place logo in the assets file
```

Comment:
this kind of makes little sense :grimacing: 

Maybe we are placing the logo in the assets directory